Mathlib/Data/Nat/Digits.lean

```diff
@@ -370,6 +373,28 @@ theorem getLast_digit_ne_zero (b : ℕ) {m : ℕ} (hm : m ≠ 0) :
     exact Nat.div_pos (le_of_not_lt hnb) (zero_lt_succ (succ b))
 #align nat.last_digit_ne_zero Nat.getLast_digit_ne_zero
 
+theorem mul_ofDigits (n : ℕ) {b : ℕ} {l : List ℕ} :
+    n * ofDigits b l = ofDigits b (l.map (n * ·)) := by
```

Comment:
It had never occurred to me that ofDigits supported unbalanced lists of digits (i.e., with numbers greater than the base), but there doesn't seem to be an issue